en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
+{name: "apple", color: "red", price: "20.0"}    
+```
 
-#### Example
+To create an empty JSON object, use the following:
 
-- Reading the Text Payload of the HTTP request.
+```ballerina
+{}
+```
 
-  ```ballerina
-  check request.getTextPayload()
-  ```
+To cast a `json` type variable `fruitDetail` to a `map<json>` type variable, use the following:
 
-### Reading the query parameter value
+```ballerina
 
-Query parameters are used in an HTTP request to send additional inputs to consider when processing the request. You can use the `request` variable's `getQueryParamValue` function to read the passed query parameter.
+<map<json>>fruitDetail
+ 
+```
+
+To access or change a field of the JSON object `fruitDetailObject`, use one of the following:
+
+!!! note
+    If you define the JSON object using a `json` type variable, you need to cast it to a `map<json>` first.
+    
+```ballerina
+fruitDetailObject["color"] = "orange"; 
+   
+fruitDetailObject["name"];
+```
+
+To create a nested JSON object, use the following:
+
+```ballerina
+    map<json> family = {
+           fname: "Alex",
+           lname: "Stallone",
+           address: {
+               line: "20 Palm Grove",
+               city: "Colombo 03",
+               country: "Sri Lanka"
+           }
+       };
+```
+
+### JSON Arrays
+
+JSON array values must be of any of the json types: `string`, `number`, `boolean`, `object`, `array`, or `null`. You can define a JSON array in one of the following two ways:
+    - Using a [`json` type variable](#simple-values-in-JSON).  
+    - Using the variable type `other` and `json[]` as the `other type`.
+You can access JSON array elements using the index.
+
+###### Examples:
+
+To declare a JSON array, use the following:
+
+```ballerina
+[1, false, null, "foo", {first: "Alex", last: "John"}] 
+```
+To cast the `json` type variable `personRecord` to a `json[]` type variable, use the following:
+
+```ballerina
+<json[]>personRecord;
+```
+
+To access elements the fourth element in the JSON array `personRecordArray`, use one of the following:
+
+!!! note
+    If you define the JSON array using a `json` type variable, you need to cast it to a `json[]` type first.
+    
+```ballerina
+personRecordArray[4]; 
+```
+
+To add a JSON array in an object literal, use the following:
+
+```ballerina
+    {
+        fname: "Alex",
+        lname: "Stallone",
+        family: [
+            {fname: "James", lname: "Stallone"},
+            {fname: "Blake", lname: "Stallone"},
+            {fname: "Kyle", lname: "Stallone"},
+            {fname: "Taylor", lname: "Stallone"}
+        ]
+    }
+```
+
+To get the length of a JSON array, use the following:
+
+```ballerina
+personRecordArray.length(); 
+```
+
+### JSON access
+
+You can use the field access(.) and optional field access (?.) to access the `json` type in Ballerina. In this case, the type checking happens at runtime. The return type of this operation is a union of `json` and `error`. 
+    
+!!! info
```

Comment:
Let's not use future tense here? 

Suggestion: " The examples in this section assume that the following variable of type `json` is already defined."

Reply:
Fixed.